kodilla-stream/src/test/java/com/kodilla/stream/sand/SandStorageTestSuite.java

```diff
@@ -0,0 +1,46 @@
+package com.kodilla.stream.sand;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+public class SandStorageTestSuite {
+    @Test
+    public void testGetSandBeansQuantity() {
+        //Given
+        List<SandStorage> continents = new ArrayList<>();
+        continents.add(new Europe());
+        continents.add(new Asia());
+        continents.add(new Africa());
+
+        //When
+        BigDecimal totalSand = BigDecimal.ZERO;
+        for(SandStorage continent : continents){
```

Comment:
stream? :)

Reply:
tutaj to nie było zadanie tylko przykład w tym submodule. W metodzie testGetSandBeansQuantityWithReduce został zastosowany stream ;)